custom_components/tahoma/config_flow.py

```diff
@@ -49,3 +49,20 @@ async def async_step_user(self, user_input=None):
         return self.async_show_form(
             step_id="user", data_schema=DATA_SCHEMA, errors=errors
         )
+
+    async def async_step_import(self, import_config: dict):
```

Comment:
Your except is not the same that the one in the above method.

Reply:
@tetienne we can not send back different error messages via `async_step_import` (for use in the front-end), thus it doesn't make sense to differentiate the exceptions.